CharlieBackend.Panel/Views/Calendar/Index.cshtml

```diff
@@ -34,11 +35,29 @@
         text-align: center;
     }
 
-    .col-1:first-child, .col-2 {
+    .col-calendar-full-week {
+        width: 100%;
+        padding-right: 0.75rem;
+        padding-left: 0.75rem;
+        flex: 0 0 14.285714%;
+        max-width: 14.285714%;
+        min-width: 100px;
+    }
+
+    .col-calendar-working-week {
+        width: 100%;
+        padding-right: 0.75rem;
+        padding-left: 0.75rem;
+        flex: 0 0 20%;
+        max-width: 20%;
+        min-width: 100px;
+    }
+
+    .col-1:first-child, .col-2, .col-calendar-full-week, .col-calendar-working-week {
         border-right: 0.5px solid;
     }
 
-    .col-2, .col-1 {
+    .col-2, .col-1, .col-calendar-full-week, .col-calendar-working-week {
         border-bottom: 0.5px solid grey;
         min-height: 100px;
     }
```

Comment:
IMHO this should be in file ~\WhatBackend\CharlieBackend.Panel\wwwroot\css\cssEdiary.css or site.css in same folder